### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231229

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1214] [Announcements]
__年前最后一次更新，提前祝大家新年快乐！__

# 系统
部分用户反馈系统升级失败，针对此问题： #1200
* 增加系统升级日志保留功能，日志保存到 `/overlay/upgrade.log`。当然，此功能只能在以后的升级生效。
* 升级失败很有可能是因为用户自行升级了系统软件包，例如`base-files`，导致系统升级功能被破坏，所以：
  1. luci “软件包”界面的“升级”按钮将遵守 opkg 的 *hold* 标志规则，被标记为 *hold* 的软件包不能被升级，将会提示错误
  2. 标记部分系统软件包，以及 iStoreOS 修改过的软件包为 *hold*， 防止被用户误升级到 OpenWRT 原版

如果之前已经升级过系统软件包，导致故障，可以从iStore安装“系统便利工具”（搜索SysTools），然后打开插件，下拉菜单选择“修复系统软件”，然后点击页面底部的“执行”按钮，完成以后会自动重启。如果重启后还有问题，可以通过恢复出厂设置来修复。

# 设备
* 通用：修复前面板网络指示灯被其他命名空间（例如 Docker 容器）的网络接口影响的问题，包括 R2S/R4S/R5S/R6S/H6xK/H28K 等等  #1146
* RK35xx: 
  1. HDMI 支持了音频输出
  2. 加回 Intel Wifi 驱动

**Comment by JIUJIUs** [comment]
周更模式！

**Comment by lizetao1314** [comment]
这个系统很好用，非常喜欢

**Comment by Amoystyle** [comment]
这个固件有问题，经常获取不到上级的DHCP，每次都得重启

**Comment by hzz-GH** [comment]
是intel的ax系列WiF芯片i可以做发射端了吗？

**Reply by jjm2473** [reply]
Intel的wifi都不支持5G AP模式

**Comment by weiwei52o** [comment]
希望天更模式

**Reply by jjm2473** [reply]
我们的固件服务器可受不了

**Comment by newstao** [comment]
ipv6通不过，通过便利工具打开也不行，不知道怎么回事

**Reply by newstao** [reply]
试验了一下，更新后，不安装便利工具，可以通ipv6，安装便利工具后反而无法连通ipv6，真的好神奇。

**Reply by jjm2473** [reply]
先配置好联网的模式（例如旁路由，PPPoE，或者二级路由），然后再执行系统便利工具里面的“开启ipv6”

**Reply by jjm2473** [reply]
ipv6有时候需要一两分钟初始化的，可能一开始不行，过一会儿就好了

**Reply by gc89925** [reply]
只有一个关闭ipv6选项，没有打开的选项啊

**Reply by jjm2473** [reply]
@gc89925  istore里面更新下啊，置顶的issue都提到了

**Comment by aluluvip** [comment]
完美的系统👍

**Comment by hujunhan22** [comment]
R4SE lan1 错误 提示网络设备不存在，好像驱动没了，怎么办。

**Reply by jjm2473** [reply]
最近没改r4s的网卡驱动啊，重启下看看

**Comment by zsnzzg** [comment]
![image](https://github.com/istoreos/istoreos/assets/112025678/91bf2013-11c1-4a69-a718-460353ff8baf)
请问这个原版在哪啊

系统便利工具选择“修复系统软件”没修复这个

**Reply by jjm2473** [reply]
这个是你自己安装的

**Reply by jjm2473** [reply]
“修复系统软件”只会修复固件自带的软件包

**Comment by francislyu0617** [comment]
提问🙋 最新的固件，反复更新十几次，没有刷写成功，之后没有任何提示。是我不配吗？
FriendlyElec NanoPi R4S，
iStoreOS 22.03.5 2023120811
